source/sass/buyers-guide/views/product.scss

```diff
@@ -84,6 +84,14 @@
   h4 {
     font-family: "Nunito Sans";
   }
+
+  @import "../creep-face-emoji";
```

Comment:
<img width="421" alt="image" src="https://user-images.githubusercontent.com/2896608/50182233-a9abd600-02c3-11e9-881d-28f0b5e4062f.png">


We need to move this `@import` line to top of the page. We shouldn't include `@import` in a declaration block as what we wanna do here is to inherit and override the existing CSS rules from `creep-face-emoji.scss` and not declare the same rules for `.product-detail .creep-face-emoji`.